### ISSUE TITLE
Python generator TypeError: Configuration.__init__() got an unexpected keyword argument 'XXX'

### ISSUE THREAD

**Issue by bayeslearner** [OpenAPITools/openapi-generator#14146]
#### Bug Report Checklist

- [X] Have you provided a full/minimal spec to reproduce the issue?
- [X] Have you validated the input using an OpenAPI validator ([example](https://apidevtools.org/swagger-parser/online/))?
- [X] Have you [tested with the latest master](https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-to-test-with-the-latest-master-of-openapi-generator) to confirm the issue still exists?
- [X] Have you searched for related issues/PRs?
- [X] What's the actual output vs expected output?

##### openapi-generator version

openapi-generator-cli 6.2.1
  commit : b0ce532
  built  : 2022-11-01T09:40:00Z
  source : https://github.com/openapitools/openapi-generator
  docs   : https://openapi-generator.tech/

##### OpenAPI declaration file content or url
https://admin.splunk.com/service/info/specs/v2/openapi.json

##### Generation Details
openapi-generator generate  -g python -o out -i https://admin.splunk.com/service/info/specs/v2/openapi.json --package-name splunkacs --skip-validate-spec

##### Steps to reproduce

It generates test code that looks like this:
```
configuration = splunkacs.Configuration(
    access_token = 'xxx')
```
Later when you run the code,  the attribute access_token is not a keyword argument of Configuration and error is thrown. 

```
 configuration = splunkacs.Configuration(
TypeError: Configuration.__init__() got an unexpected keyword argument 'access_token'
```

This is strange as Configuration class does have `__set_attr__' defined. 

##### Suggest a fix
I'm assuming that Configuration class is supposed to use __set__attr__ to catch undefined keyword argument, why didn't it work?

**Comment by spacether** [comment]
FYI this was fixed earlier in the future home pf the python generator: https://github.com/openapi-json-schema-tools/openapi-json-schema-generator/commit/84557a98244ae020568afa7781bf253d40abf5f7
and is available in the v1.0.3 release there

**Comment by spacether** [comment]
@bayeslearner fixed
